src/SayMore/UI/ConvertMediaDlgViewModel.cs

```diff
@@ -131,7 +141,13 @@ public string GetNewOutputFileName(bool returnFileNameOnly)
 			}
 
 			if (outputFile == null)
-				outputFile = GetNewOutputFileName(false);
+			{
+				if (!string.IsNullOrEmpty(_outputFile))
```

Comment:
This check here seems to be redundant, since GetNewOutputFilename already does this.